src/components/MyAccount/RequestsTab/RequestsTab.test.tsx

```diff
@@ -229,4 +232,31 @@ describe("RequestsTab", () => {
     freezeButtons = component.getAllByText("Freeze")
     expect(freezeButtons.length).toBe(1)
   })
+
+  it("shows pick up by date and status when circ request is ready", () => {
+    const component = render(
+      <RequestsTab
+        patron={mockPatron}
+        holds={mockHolds}
+        removeHold={mockRemoveHold}
+      />
+    )
+    const readyCircRequestRow = component.getAllByRole("row")[5]
+    expect(readyCircRequestRow).toHaveTextContent("May 15, 2024")
+    expect(readyCircRequestRow).toHaveTextContent("READY FOR PICKUP")
+  })
+  it("does not show freeze button on freezable request when it is anything other than pending", () => {
+    const component = render(
+      <RequestsTab
+        patron={mockPatron}
+        holds={mockHolds}
+        removeHold={mockRemoveHold}
+      />
+    )
+    const readyCircRequestRow = component.getAllByRole("row")[5]
```

Comment:
Nitpick: The variable name hints at it but I think it'll be good to show/test that "Ready for pickup" is displayed for this row as well.

Reply:
Same for the row below.